templates/default/get_transit_tiles.sh.erb

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+# make sure only one is running at any time...
+LOCK_FILE="<%= node[:valhalla][:lock_dir] %>/cut_transit_tiles.lock"
+(set -C; : > ${LOCK_FILE}) 2> /dev/null
+if [ $? != "0" ]; then
+   echo "Lock file exists"
+   exit 0
+fi
+trap 'rm $LOCK_FILE' EXIT 1 2 3
+
+export PATH=$PATH:/usr/local/bin
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
+
+# TODO: for incremental updates we'll want to first get a copy of the previous stuff
+#if [ -f <%= node[:valhalla][:transit_dir] %>.tgz ]; then
+#  rm -rf <%= node[:valhalla][:transit_dir] %>_temp
+#  mkdir <%= node[:valhalla][:transit_dir] %>_temp
+#  tar pxvf <%= node[:valhalla][:transit_dir] %>.tgz -C <%= node[:valhalla][:transit_dir] %>_temp
+#fi
+
+#tar pxvf -C <%= node[:valhalla][:transit_dir] %>_temp <%= node[:valhalla][:transit_dir] %>.tgz
+
+# fetch the latest data
+transit_fetcher <%= node[:valhalla][:conf_dir] %> http://transit.land 5000 <%= node[:valhalla][:transit_dir] %>_temp <%= node[:valhalla][:transit_api_key] %>
```

Comment:
i considered that but didnt think there was a real use case for hitting dev, if we think there is, i can change this to be configurable (but i also wasnt really keen on putting dev urls out in the wild)